index.html

```diff
@@ -0,0 +1,317 @@
+              <li>HTML</li>
+              <li>SASS/CSS</li>
+              <li>JavaScript</li>
+              <li>WebPack</li>
+              <li>Babel</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/Weather-App/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card conference">
+          <div class="card-description">
+            <h2>Conference page</h2>
+            <p class="white-description">
+              This is a sample conference page made using HTML and CSS.
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>SASS</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/Conference-page/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card katchy">
+          <div class="card-description">
+            <h2>Katchy</h2>
+            <p class="white-description">
+              This is a website for recipes made with Ruby on Rails
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>Ruby</li>
+              <li>Ruby on Rails</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>ERB</li>
+              <li>RSPec</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/katchy/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
```

Comment:
It will be good to use the `section` tag when creating a new section. This will ensure that your application is more semantic. You can learn more about semantic tags using this [article](https://www.w3schools.com/html/html5_semantic_elements.asp)